mccoy/workflow/rules/beast.smk

```diff
@@ -42,6 +44,9 @@ def beast_params(wildcards):
     return " ".join(params)
 
 
+print(config['beast'])
```

Comment:
Is this debugging code? If so, can it be removed?

Reply:
yes - sorry about that :(

Reply:
I've removed that now